kernel/src/process.rs

```diff
@@ -694,8 +712,8 @@ impl<S: UserspaceKernelBoundary> ProcessType for Process<'a, S> {
         (self.mem_end() as *mut *mut u8).offset(-(grant_num + 1))
     }
 
-    fn get_process_name(&self) -> &[u8] {
-        self.process_name.as_bytes()
+    fn get_process_name(&self) -> &'static str {
```

Comment:
I've been wary of pulling in str when working on the ProcessConsole because of the possible code pull-in; does making this change significantly increase code size?